egs/reazonspeech/ASR/RESULTS.md

```diff
@@ -47,3 +47,41 @@ The decoding command is:
     --blank-penalty 0
 ```
 
+#### Streaming
+
+We have not completed evaluation of our models yet and will add evaluation results here once it's completed.
+
+The training command is:
+```shell
+./zipformer/train.py \
+  --world-size 8 \
+  --num-epochs 40 \
+  --start-epoch 1 \
+  --use-fp16 1 \
+  --exp-dir zipformer/exp-large \
+  --causal 1 \
+  --num-encoder-layers 2,2,4,5,4,2 \
+  --feedforward-dim 512,768,1536,2048,1536,768 \
+  --encoder-dim 192,256,512,768,512,256 \
+  --encoder-unmasked-dim 192,192,256,320,256,192 \
+  --lang data/lang_char \
+  --max-duration 1600
+```
+
+The decoding command is:
+
+```shell
+./zipformer/streaming_decode.py \
+  --epoch 28 \
```

Comment:
Since the Reazonspeech is a large dataset, I suggest that you replace `--epoch` with `--iter`. See also `RESULTS.md` from our Gigaspeech recipe. You can find example usages of `--iter` there.